chirp/wxui/bankedit.py

```diff
@@ -222,6 +222,12 @@ def _index_changed(self, event):
         value = event.GetString()
         if isinstance(self._col_defs[col], ChirpBankIndexColumn):
             self._change_memory_index(self.row2mem(row), int(value))
+        # if activating a checkbox with spacebar, treat it like a click
+        elif isinstance(self._col_defs[col], ChirpBankToggleColumn):
```

Comment:
The name of this method (`_index_changed`) is no longer correct with this change as it's handling more than just a changing index. Can you rename it so it's more obvious?